spec/helpers/administrate/application_helper_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "rails_helper"
+
+RSpec.describe Administrate::ApplicationHelper do
+  describe "#display_resource_name" do
+    it "defaults to the plural of the model name" do
+      displayed = display_resource_name(:customer)
+
+      expect(displayed).to eq("Customers")
+    end
+
+    it "handles string arguments" do
+      displayed = display_resource_name("customer")
+
+      expect(displayed).to eq("Customers")
+    end
+
+    it "handles plural arguments" do
+      displayed = display_resource_name(:customers)
+
+      expect(displayed).to eq("Customers")
+    end
+
+    context "when translations are defined" do
+      it "uses the plural of the defined translation" do
+        translations = {
+          activerecord: {
+            models: {
+              customer: {
+                one: "User",
+                other: "Users",
+              } } } }
```

Comment:
@mcmire I originally had them all on their own line, couldn't decide which I liked better. If you like them on their own line, I can revert those commits.

Each on their own line: https://github.com/thoughtbot/administrate/commit/2250b3d43283737a32242dbf85302dc2957dfaf7#diff-8f8d41bf7588d283cc64bf1d2ef099dcR22